.circleci/config.yml

```diff
@@ -1353,24 +1353,23 @@ workflows:
           requires:
             - hold
       - make-release:
-          <<: *release-tags
```

Comment:
removing the filters and adding the requirement should mean that this runs when the hold step finishes